kornia/augmentation/_2d/intensity/planckian_jitter.py

```diff
@@ -66,7 +66,7 @@
 
 
 class PlanckianJitter(IntensityAugmentationBase2D):
-    r"""Apply planckian jitter transformation to input tensor
+    r"""Apply planckian jitter transformation to input tensor.
```

Comment:
Can you add new line here ? We follow the docs convention of having one first line with very brief description follows by more technical things, args, etc

Reply:
Added.